config/runtime.exs

```diff
@@ -6,8 +6,8 @@ config :realtime_signs, RealtimeSignsWeb.Endpoint,
   server: true
 
 config :realtime_signs,
-  sign_head_end_host: System.get_env("SIGN_HEAD_END_HOST", "127.0.0.1"),
-  sign_ui_url: System.get_env("SIGN_UI_URL", "signs-dev.mbtace.com"),
+  sign_head_end_host: System.get_env("SIGN_HEAD_END_HOST", Application.get_env(:realtime_signs, :sign_head_end_host)),
+  sign_ui_url: System.get_env("SIGN_UI_URL", Application.get_env(:realtime_signs, :sign_ui_url)),
```

Comment:
Is the thought here to have `sign_head_end_host` and `sign_ui_url` default to `nil` whenever they aren't set in an env variable? If so, we can omit the call to `Application.get_env/2` and just not include a default value since `System.get_env/1` will automatically return `nil` if it doesn't find the env var that was requested.

Reply:
That `get_env` is there for test mode, where we want network calls to be "enabled", even though the actual HTTP module is mocked. The test config sets up dummy hostnames, and this code makes sure that's retained instead of overwritten by the lack of an environment variable.

Reply:
Ahh gotcha that makes sense!